### PR TITLE
README update

### PR DESCRIPTION
**Pull request by LissaGreense**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,168 @@
...
+
+**watermoviemelon** is a fun, interactive web application designed to bring friends, family, and movie lovers together
+to organize unforgettable movie nights! It makes planning and hosting movie events a breeze. Users can create a movie
+night, add movie options, and let the app randomly select a film for the evening—keeping the suspense alive until
+showtime! Once the night ends, everyone can rate the film!
+
+### Key Features:
+
+- **Event Creation**: Easily set up movie nights or join to the existing one.
+- **Movie List Management**: Add movCheck out how many movie nights you’ve hosted, how , and flex a little among your
```
Comment (adlbrtbsr) [inline README.md:15]: whats movCheck

File: README.md [hunk 2]
```diff
@@ -1,11 +1,168 @@
...
+  friends.ies to a global list shared across all movie nights. Once a film is added, it remains
```
Comment (adlbrtbsr) [inline README.md:16]: friends.ies? what do you mean?

File: README.md [hunk 3]
```diff
@@ -1,11 +1,168 @@
...
+
+Before running the app, ensure that you have the following installed:
+
+- **Docker**: Required for running the application using Docker
+  containers. [Get Docker](https://docs.docker.com/engine/install/)
+- **Docker Compose**: Used to manage multi-container
+  applications. [Get Docker Compose](https://docs.docker.com/compose/install/)
+- **Node.js**: For the frontend to work locally. [Get Node.js](https://nodejs.org/en/download/package-manager)
+- **Python 3.x**: Required for the backend. [Get Python](https://www.python.org/downloads/)
+-
```
Comment (adlbrtbsr) [inline README.md:62]: empty listing

File: README.md [hunk 4]
```diff
@@ -1,11 +1,168 @@
...
+npm run dev
+```
+
+Or with yarn:
+
+```bash
+yarn dev
+```
+
+4. The app should now be running locally at http://localhost:3000. You can make changes to the source code, and the
```
Comment (adlbrtbsr) [inline README.md:107]: i dont think frontend dev runs on :3000 by default

File: README.md [hunk 5]
```diff
@@ -1,11 +1,168 @@
...
+
+TODO
+
 ## Authors
+
 - [LissaGreense](https://github.com/LissaGreense)
 - [adlbrtbsr](https://github.com/adlbrtbsr)
 - [kajedot](https://github.com/kajedot)
 
 ## How to run
```
Comment (adlbrtbsr) [inline README.md:164]: i dont think this section is needed anymore

File: README.md (hunk 3 above)
Comment (LissaGreense) [reply README.md]: fixed ;)

File: README.md (hunk 2 above)
Comment (LissaGreense) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (LissaGreense) [reply README.md]: sometimes it's better to know less. Fixed ;)

File: README.md (hunk 4 above)
Comment (LissaGreense) [reply README.md]: u r right ;) I replaced with the real one

**Review by adlbrtbsr — APPROVED** [review]
good job

**Pull request merged**